scripts/in_container/verify_providers.py

```diff
@@ -826,6 +828,12 @@ def run_provider_discovery():
     subprocess.run(["airflow", "providers", "secrets"], check=True)
     console.print("[bright_blue]List all auth backends[/]\n")
     subprocess.run(["airflow", "providers", "auth"], check=True)
+    if version > "2.6.0":
```

Comment:
This comparison is technically wrong. String comparison does not work for package versions. For example 2.10.0 will be less than 2.6.0. see example of comparison i do in https://github.com/apache/airflow/pull/30994 for good comparision example.

Reply:
updated, thanks for pointing out